tests/test_cmdline.py

```diff
@@ -713,22 +688,17 @@ def test__no_options__defaults_to_check_action_in_current_dir(
     )
 
     expect = [
+        "",
         "These imports appear to be undeclared dependencies:",
         "- 'requests' imported at:",
-        "    code.py:1",
+        f"    {tmp_path / 'code.py'}:1",
         "",
         "These dependencies appear to be unused (i.e. not imported):",
         "- 'pandas' declared in:",
-        "    requirements.txt",
-    ]
-    expect_logs = [
-        "INFO:fawltydeps.extract_imports:Parsing Python files under .",
-        "INFO:fawltydeps.packages:'pandas' was not resolved."
-        " Assuming it can be imported as 'pandas'.",
+        f"    {tmp_path / 'requirements.txt'}",
```

Comment:
I'm a little worried about these changes. As I understand it, where `run_fawltydeps()` can use the `cwd` to actually set the current working directory for the subprocess, `run_fawltydeps_functon()` cannot do the same, since everything happens in the same process. Instead, you work around that by passing `cwd` as a `basepath`/positional argument instead, and the consequence of that is to adjust the expected output here.

I would like to keep test coverage of the fact that when we work on paths relative to cwd, then that is reflected in the output.

I wonder if we can use [`monkeypatch.chdir`](https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.chdir) to temporarily change cwd while running the tests? However, it probably means changing `run_fawltydeps_function()` into a pytest fixture (to have access to `monkeypatch`).

Alternatively, kick this to a later PR, and leave it as `run_fawltydeps()` instead of `run_fawltydeps_function()` for now.

Reply:
Created #302 not to forget.